modules.d/00systemd/module-setup.sh

```diff
@@ -23,10 +23,8 @@ installkernel() {
 
 # called by dracut
 install() {
-    local _mods
-
-    if [[ $prefix == /run/* ]]; then
```

Comment:
This is the change that seem to break tests

Reply:
Yeah; fixed